Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php

```diff
@@ -110,6 +112,27 @@ protected function getRecords(): array {
         // Without sql renderer the query would fail. The actual value is fetched afterward.
         'sql_renderer' => fn () => '(SELECT NULL)',
       ],
+      [
+        'name' => 'application_process_progress',
```

Comment:
I'd name it `application_process_review_progress`.

Reply:
will change